web/src/components/planner/PlannerFactory.vue

```diff
@@ -111,13 +119,13 @@
                   />
                 </span>
                 <span>
-                  <b>{{ getPartDisplayName(part.id) }}</b>: {{ formatNumber(part.amount) }}/min
```

Comment:
Not sure why these `formatNumber`s were reverted?

Reply:
Ah, that was a mistake during forking. Fixed!